module/signature/invalid_threshold.go

```diff
@@ -0,0 +1,32 @@
+// +build relic
+
+package signature
+
+import "github.com/onflow/flow-go/crypto"
+
+// This implementation of `ThresholdProvider` always produces invalid signatures
+// (that are inexpensively detectable as invalid) as we cannot produce valid signatures.
+// Used when this node fails to generate a random beacon key during the last DKG.
+// This will be replaced by consensus voting v2, which will be able to gracefully
+// handle consensus nodes submitting votes with only one signature.
+type InvalidThresholdProvider struct {
+	*ThresholdVerifier
+}
+
+// NewInvalidThresholdProvider returns a new instance of `InvalidThresholdProvider`
+func NewInvalidThresholdProvider(tag string) *InvalidThresholdProvider {
+	tp := &InvalidThresholdProvider{
+		ThresholdVerifier: NewThresholdVerifier(tag),
+	}
+	return tp
+}
+
+// Sign always returns an invalid signature
+func (tp *InvalidThresholdProvider) Sign(msg []byte) (crypto.Signature, error) {
+	return crypto.BLSInvalidSignature(), nil
```

Comment:
Since we are producing a invalid signature here for this specific case, it's better to treat this error separately, in order to distinguish from real exceptions.

We could handle the BLS invalid signature error [here, and log as warning](https://github.com/onflow/flow-go/blob/master/consensus/hotstuff/eventhandler/event_handler.go#L126-L131)

Reply:
@zhangchiqing can't we treat a proposal with an invalid signature like any other wrong proposal? (slashing is not implemented yet anyway)

Reply:
we could. 

Just that I think there supposed to be no wrong proposal with invalid signature, if there is, then probably a bug. If we treat this as an error, then it's hard to distinguish from a real error.